docs/source/getting_started/quickstart/customer_choice/index.md

```diff
@@ -1,5 +1,41 @@
 # Customer Choice Quickstart
 
-TODO - add python code block below with minimal code to get started with customer choice modeling
+```python
+from pymc_marketing.customer_choice.mv_its import (
```

Comment:
Can we import in customer_choice/__init__ so we can use
`from pymc_marketing.customer_choice import MVITS, ...`

Reply:
done :)